proto/iotics/api/meta.proto

```diff
@@ -38,6 +38,30 @@ service MetaAPI {
   rpc SparqlUpdate(SparqlUpdateRequest) returns (SparqlUpdateResponse) {}
 }
 
+// ExplorerAPI - Deprecated. Use MetaAPI instead.
+service ExplorerAPI {
```

Comment:
Could it go in the MetaAPI ? (less changes for the integration and a bit more inline with https://github.com/uber/prototool/blob/dev/docs/lint.md `SERVICE_NAMES_MATCH_FILE_NAME`)

Reply:
👍